### PR TITLE
Change replicate command cli

### PR DESCRIPTION
**Pull request by poornas**

### CODE REVIEW

File: cmd/admin-bucket-remote.go [hunk 1]
```diff
@@ -18,22 +18,22 @@ package cmd
 
 import "github.com/minio/cli"
 
-var adminBucketReplicationCmd = cli.Command{
-	Name:   "replication",
-	Usage:  "Manage replication targets",
-	Action: mainAdminBucketReplication,
+var adminBucketRemoteCmd = cli.Command{
+	Name:   "remote",
+	Usage:  "Manage bucket targets",
```
Comment (harshavardhana) [inline cmd/admin-bucket-remote.go:23]:
```suggestion
	Usage:  "manage remote bucket targets",
```

File: cmd/replicate-add.go [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+	  
+USAGE:
+	{{.HelpName}} TARGET
+	  
+FLAGS:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+EXAMPLES:
+	1. Add replication configuration rule on bucket "mybucket" for alias "myminio".
+	   {{.Prompt}} {{.HelpName}} myminio/mybucket/prefix --tags "key1=value1&key2=value2" --storage-class "STANDARD" --arn 'arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket' --priority 1
```
Comment (harshavardhana) [inline cmd/replicate-add.go:74]:
```suggestion
	   {{.Prompt}} {{.HelpName}} myminio/mybucket/prefix --tags "key1=value1&key2=value2" \
	               --storage-class "STANDARD" \
	               --arn 'arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket' --priority 1
```

File: cmd/replicate-add.go [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+	
+	2. Disable replication configuration rule with rule Id "bsibgh8t874dnjst8hkg" and Arn "arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket" on bucket "mybucket" for alias "myminio".
+	   {{.Prompt}} {{.HelpName}} myminio/mybucket/prefix --id "bsibgh8t874dnjst8hkg" --disable --arn 'arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket'
```
Comment (harshavardhana) [inline cmd/replicate-add.go:77]:
```suggestion
	   {{.Prompt}} {{.HelpName}} myminio/mybucket/prefix --id "bsibgh8t874dnjst8hkg" \
	              --disable --arn 'arn:minio:replica::c5be6b16-769d-432a-9ef1-4567081f3566:destbucket'
```

File: docs/minio-admin-complete-guide.md [hunk 4]
```diff
@@ -893,22 +893,22 @@ mc admin bucket quota myminio/mybucket --hard 64MB
 mc admin bucket quota myminio/mybucket --clear
 ```
 
-<a name="replication"></a>
-### Command `replication` - Set/Get/Remove bucket replication target
-`replication` command to set or get bucket replication target on MinIO server.
+<a name="remote"></a>
+### Command `remote` - Manage bucket targets
```
Comment (harshavardhana) [inline docs/minio-admin-complete-guide.md:897]:
```suggestion
### Command `remote` - manage bucket targets
```

File: docs/minio-admin-complete-guide.md [hunk 5]
```diff
@@ -893,22 +893,22 @@ mc admin bucket quota myminio/mybucket --hard 64MB
...
+  mc admin bucket remote - manage bucket targets
 
 USAGE:
-  mc admin bucket replication set TARGET http(s)://ACCESSKEY:SECRETKEY@REPLICA_URL/REPLICA_BUCKET [--path | --api]
+  mc admin bucket remote set TARGET http(s)://ACCESSKEY:SECRETKEY@TARGET_URL/TARGET_BUCKET [--path | --api] --type
 
-REPLICA_BUCKET:
-  Also called as replication target bucket.
+TARGET_BUCKET:
+  Also called as target bucket.
```
Comment (harshavardhana) [inline docs/minio-admin-complete-guide.md:908]:
```suggestion
  Also called as remote bucket.
```

File: cmd/bucket-main.go [hunk 6]
```diff
@@ -0,0 +1,43 @@
+/*
+ * MinIO Client (C) 2020 MinIO, Inc.
```
Comment (kannappanr) [inline cmd/bucket-main.go:2]: why do we need this file? There are no `bucket` commands

Comment (poornas) [reply cmd/bucket-main.go:2]: because ILM and lock are not merged yet

File: cmd/main.go [hunk 7]
```diff
@@ -347,6 +347,8 @@ var appCmds = []cli.Command{
 	watchCmd,
 	policyCmd,
 	tagCmd,
+	versioningCmd,
```
Comment (kannappanr) [inline cmd/main.go:350]:
```suggestion
```

**Pull request merged**